package.json

```diff
@@ -115,6 +115,7 @@
     "aws-sdk": "^2.451.0",
     "bootstrap": "^3.3.6",
     "chroma-js": "^2.1.0",
+    "eslint-plugin-cypress": "^2.10.3",
```

Comment:
Was this intended to be in this PR?

Reply:
As before, this should be in `devDependencies` not `dependencies`. As a result of this change it's now in both.

Reply:
Removed